simplistic navbar

Lunchii opened the pull request:
Made the nav bar no longer transparent with a black/dark gray background. It doesn't overlap with the body/content. Changed the text font (random one for now). On mobile, I just decided to go for a navbar that doesn't take up the entire screen but instead expands it downwards/vertically. 

Also the other "changes" are just commenting stuff out so the only real change is in the nav bar component.

Cheekane commented:
@Lunchii You should probably uncomment the login code.

Cheekane commented:
> What are your thoughts on adding more padding above and below and also making the logo bigger?

It probably shouldn't be that big.

@Lunchii Here are things that need to be fixed and changed.
1. The navbar color theme could be picked to be nicer color. Experiment with some colors. Or maybe gray is fine.
2. The navbar should transform the page navigation options to a hamburger menu.
3. You should adjust the font and proportions if you can make it look nicer.

Lunchii commented:
1. Changed the font and font size and made it semibolded (will change it back if it doesn't look good)
2. Mobile Navbar can be expanded and can be closed if clicked outside of the panel
3. Removed the sidebar

We can probably change the navbar/font colour to something else

Cheekane reviewed (approved):
@Lunchii Good looks
We will be changing the color and custom styling a little later. But this is good for now.

jerpyk reviewed (commented):
<img width="690" alt="Screenshot 2025-06-10 at 5 06 31 PM" src="https://github.com/user-attachments/assets/235d2c6a-3a93-488a-8d2d-fe14e793c1b5" />
Is it broken 😑

Cheekane reviewed (approved):
🙅
